### ISSUE TITLE
tinytag unable to process valid mp3

### ISSUE THREAD

**Issue by martmists** [devsnd/tinytag#35]
I was trying to use `tinytag.TinyTag.get(filename)`, but it errored and said 
`tinytag.tinytag.TinyTagException: mp3 parsing failed`.
Now I get that this means the mp3 is invalid, but the weird thing is: 
It works with ffplay, VLC and WMP (and probably all music playing software)

File was downloaded using youtube_dl, not sure if that matters.

```
Traceback (most recent call last):
  File "D:\Program_Files\Python\lib\site-packages\tinytag\tinytag.py", line 311, in _determine_duration
    frame_bitrate = ID3.bitrate_by_version_by_layer[mpeg_id][layer_id][br_id]
TypeError: 'NoneType' object is not subscriptable

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "D:\Program_Files\Python\lib\site-packages\tinytag\tinytag.py", line 94, in get
    tag.load(tags=tags, duration=duration, image=image)
  File "D:\Program_Files\Python\lib\site-packages\tinytag\tinytag.py", line 115, in load
    self._determine_duration(self._filehandler)
  File "D:\Program_Files\Python\lib\site-packages\tinytag\tinytag.py", line 313, in _determine_duration
    raise TinyTagException('mp3 parsing failed')
tinytag.tinytag.TinyTagException: mp3 parsing failed
```

**Comment by devsnd** [comment]
Hey @martmists!

Thanks for your report. Can you please provide the file by sending it to my email address, tomwallroth [aŧ] gmail [døt] com

Without the file it will be hard to reproduce the problem. Once I have it, I can have a look at it.

**Comment by kurdyak** [comment]
Hi, I have the same error using tinytag.TinyTag.get(filename)

`Traceback (most recent call last):
  File "test.py", line 127, in <module>
    tag = TinyTag.get('/tmp/temp.mp3')
  File "/usr/local/lib/python2.7/dist-packages/tinytag/tinytag.py", line 105, in get
    tag.load(tags=tags, duration=duration, image=image)
  File "/usr/local/lib/python2.7/dist-packages/tinytag/tinytag.py", line 126, in load
    self._determine_duration(self._filehandler)
  File "/usr/local/lib/python2.7/dist-packages/tinytag/tinytag.py", line 504, in _determine_duration
    raise TinyTagException('mp3 parsing failed')
tinytag.tinytag.TinyTagException: mp3 parsing failed`

If you need more info or file, which call the error - I'll give it to you.

**Comment by devsnd** [comment]
Hey @kurdyak,

please send the failing file to the email address as described above, thanks.

**Comment by kurdyak** [comment]
Thank you for your replay!
I've sent you the file to your email.

**Comment by findix** [comment]
I got this problem too. I will send the files to you.
Thank you for your code and sharing :)

**Comment by krystalize** [comment]
Hey, I am getting this error. Was there ever a resolution to this?

**Comment by diatomee** [comment]
Hey, I have the same problem with some files. I also met him with [music-metadata](https://www.npmjs.com/package/music-metadata) (if it helps).

**Comment by westcoj** [comment]
Not sure if you looked into it or not. I noticed the issue was occurring when trying to check the duration, specifically on lines 507-510. I looked at the audio files (mp3) that were throwing this but couldn't really make any connections between them that didn't exist between working files. If you specify duration as false all the files work, though if someone needs the duration I guess that doesn't do them any good. I didn't look into it much more since I don't need the duration, but figured I'd post here.
